usr/share/rear/prep/COVE/default/390_check_cove.sh

```diff
@@ -0,0 +1,13 @@
+#
+# Check if the Backup Manager is installed
+#
+
+if [ -z "$COVE_INSTALL_DIR" ]; then
+    Error "COVE_INSTALL_DIR cannot be empty. Please define it in local.conf."
```

Comment:
```suggestion
    Error "COVE_INSTALL_DIR cannot be empty. Please define it in your configuration."
```

Better not to point them to a specific configuration file.